14-tools/README.md

```diff
@@ -0,0 +1,32 @@
+## Tools
+
+- Sonarqube
+
+![](../images/Sonarqube.png)
+
+- CS fixers
+
+![](../images/cs-fixer.png)
+
+- CI/CD
+
+![](../images/ci-cd.jpg)
+
+- Code Reviews
+
+![](../images/code-review.jpeg)
+
+### Speaker Notes
+- Sonarqube
+    - It's an open-source code quality inspection tool.
+    - It can  perform automatic reviews with static analysis of code to detect bugs, code smells, and security vulnerabilities.
+- CS fixers
```

Comment:
Make this as examples, as the discussion is not based on PHP, we have cs fixers for other languages.

Also, we can make use of the IDE, git hooks also to get these done.

Reply:
CS is not only for PHP. this stands for coding standard fixer. I added some examples as well